app/views/validators/_block_history_chart.html.erb

```diff
@@ -9,31 +9,50 @@
 
     // The data for our dataset
     data: {
-        // x-axis labels
-        labels: <%= @data.keys %>,
-        // An Array of JSON hashes for each dataset to plot. The y-axis is
-        // dynamicly calculated. Look for othe options to control the y-axis if
-        // desired.
-        datasets: [
-          {
-            label: 'Skip %',
-            fill: false,
-            backgroundColor: 'Green',
-            borderColor: 'Green',
-            borderWidth: 1,
-            radius: 0,
-            data: <%= @data.map { |k,v| v[:skipped_slot_percent].round(1) }%>
-          },
-          {
-            label: 'Cluster %',
-            fill: false,
-            backgroundColor: 'Blue',
-            borderColor: 'Blue',
-            borderWidth: 1,
-            radius: 0,
-            data: <%= @data.map { |k,v| v[:cluster_skipped_slot_percent].round(1) } %>
-          }
-        ]
+      // x-axis labels
+      labels: <%= @data.keys %>,
+      // An Array of JSON hashes for each dataset to plot. The y-axis is
+      // dynamicly calculated. Look for othe options to control the y-axis if
+      // desired.
+      datasets: [
+        {
+          label: 'Skip %',
```

Comment:
Dotted line in a lighter color?

Reply:
Here's what it looks like w/ that change @brianlong 

![image](https://user-images.githubusercontent.com/5379519/112392673-6a009f80-8cbf-11eb-8f35-e26a01470d37.png)

Reply:
Looks good